packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -23,10 +44,37 @@ export const ActivityLibrary = (props: Props) => {
   const data = usePreloadedQuery<ActivityLibraryQuery>(query, queryRef, {
     UNSTABLE_renderPolicy: 'full'
   })
+  const {viewer} = data
+  const {featureFlags, availableTemplates} = viewer
+
+  const templates = [
+    {id: 'action', type: 'action', name: 'Check-in', team: {name: 'Parabol'}},
+    {id: 'teamPrompt', type: 'teamPrompt', name: 'Standup', team: {name: 'Parabol'}},
+    ...availableTemplates.edges.map((edge) => edge.node)
+  ] as {
```

Comment:
+1 to have a better typesafety in this case it'd be better to specify type in the variable declaration
```
const templates: {
    id: string
    name: string
    type: MeetingTypeEnum
    team: {name: string}
  }[] = ...
```

as what casting does is basically saying, `not gonna check anything, I'll assume the structure is correct`